templates/shared/forms/interactive/storage.html

```diff
@@ -2,117 +2,75 @@
-              <h3>Your comments</h3>
-              <ul class="p-list">
-                <li class="p-list__item">
-                  <label for="Comments_from_lead__c">What would you like to talk to us about?</label>
-                  <textarea id="Comments_from_lead__c" name="Comments_from_lead__c" rows="5"
-                    maxlength="2000"></textarea>
-                </li>
-              </ul>
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="returnURL"
-                value="%% returnURL %%" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formid" value="%% formid %%" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Consent_to_Processing__c"
-                value="yes" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_campaign" id="utm_campaign"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_medium" id="utm_medium"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_source" id="utm_source"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_content" id="utm_content"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="utm_term" id="utm_term"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="GCLID__c" id="GCLID__c"
-                value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" name="Facebook_Click_ID__c"
-                id="Facebook_Click_ID__c" value="" />
-              <input type="hidden" aria-hidden="true" aria-label="hidden field" id="preferredLanguage"
-                name="preferredLanguage" maxlength="255" value="" />
-            </div>
-          </div>
-
-          <div class="u-sv2">By submitting this form, I confirm that I have read and agree to <a
+      <form action="/marketo/submit" method="post" id="mktoForm_%% formid %%" class="modal-form">
+        <div class="row">
+          <ul class="p-list col-6">
+            <li class="p-list__item">
+              <label for="firstName">First name:&ast;</label>
+              <input id="firstName" name="firstName" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <label for="lastName">Last name:&ast;</label>
+              <input id="lastName" name="lastName" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <label for="email">Email:&ast;</label>
+              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"
+                required="required" />
```

Comment:
```suggestion
              <label for="email" classs="is-required">Email:</label>
              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$"
                required />
```
This is already set to true as is, no need to assign a value